projects/superhero-dex/index.js

```diff
@@ -0,0 +1,29 @@
+const { get } = require('../helper/http')
+const { transformDexBalances } = require("../helper/portedTokens");
+
+async function tvl(api) {
+  const allPools = [];
+  const pairs = await get('https://dex-backend-mainnet.prd.service.aepps.com/pairs')
+  for (const pair of pairs) {
+    const [pairLiquidityInfo] = await get('https://dex-backend-mainnet.prd.service.aepps.com/history?pairAddress=' + pair.address + '&limit=100&order=desc&toBlockTime=' + api.timestamp * 1000);
+    allPools.push({
+      token0: pair.token0,
+      token0Bal: pairLiquidityInfo?.reserve0 || 0,
+      token1: pair.token1,
+      token1Bal: pairLiquidityInfo?.reserve1 || 0,
+    })
+  }
+
+  return transformDexBalances({
+    chain: 'aeternity',
+    data: allPools,
+  });
+}
+
+
+module.exports = {
```

Comment:
can you add `timetravel: false, ` to notify that this cant be backfilled

Reply:
I will fix it

Reply:
But it can be backfilled. I could not find a precise documentation but I assumed the backfilling to happen through api.timestamp?

Reply:
my bad, missed the part where timestamp is sent as part of request, removing timetravel flag.

Reply:
Thanks!